src/components/teasers/teaser.tsx

```diff
@@ -42,7 +37,9 @@ const ToplineContainer = styled.div`
   justify-content: space-between;
 `;
 
-const TeaserText = styled(Text)`
+const TeaserText = styled.div`
+  font-size: 16px;
+  line-height: 150%;
```

Comment:
Does this match any of the existing typography configurations? Let's try to re-use them whenever possible :pray:

Reply:
We need to refactor the teaser story, I would prefer to handle it there. I just copied over the styles of the Text component which is an outdated component.